decidim/decidim
# Redesign: responsive links on cards

#### :tophat: What? Why?
Make links responsive on cards

#### :pushpin: Related Issues
- Fixes #11361 

#### Testing
1. Go to a process with the blog content block on the landing page and a block with a long link,
2. See the link is fully inside the card,

### :camera: Screenshots
https://github.com/decidim/decidim/assets/6973564/5fa82f2d-2b66-4aea-98b2-135bf5270b33

:hearts: Thank you!
